CMakeLists.txt

```diff
@@ -106,9 +106,11 @@ IF(NOT LIBTYPE)
 ENDIF()
 
 SET(LIB_MAJOR_VERSION "1")
-SET(LIB_MINOR_VERSION "19")
+SET(LIB_MINOR_VERSIONA "1")
+SET(LIB_MINOR_VERSIONB "9")
 SET(LIB_REVISION "1")
-SET(LIB_VERSION "${LIB_MAJOR_VERSION}.${LIB_MINOR_VERSION}.${LIB_REVISION}")
+SET(LIB_VERSION "${LIB_MAJOR_VERSION}.${LIB_MINOR_VERSIONA}${LIB_MINOR_VERSIONB}.${LIB_REVISION}")
+SET(LIB_VERSION_NUM ${LIB_MAJOR_VERSION},${LIB_MINOR_VERSIONA},${LIB_MINOR_VERSIONB},${LIB_REVISION})
```

Comment:
Why does this split `19` into `1,9`?

Reply:
`FILEVERSION` and `PRODUCTVERSION` use a four-part numbering scheme; If the fourth is not specified the value zero is implicitly substituted. The other `FileVersion` and `ProductVersion` variables however are strings and do not have this quirk.

This was simply to make it look nicer when viewed, but is not necessary for functionality.

Reply:
I'd prefer to keep it as the three-number scheme, to avoid confusion (`1.19.0` is quite different from `1.1.9.0`, same as `1.20.0` and `1.2.0.0`). Appending an extra `0` for the fourth value is fine.

Reply:
I have pushed a new commit that reverts the changes.